File: src/main/java/no/nav/melosys/eessi/models/sed/nav/Arbeidssted.java

package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;
import lombok.Data;

@JsonInclude(Include.NON_NULL)
@Data
public class Arbeidssted {

    private Adresse adresse;

    private String erikkefastadresse;

    private String hjemmebase;

    private String navn;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/Arbeidssted.kt
package no.nav.melosys.eessi.models.sed.nav

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class Arbeidssted(
    var adresse: Adresse? = null,
    var erikkefastadresse: String? = null,
    var hjemmebase: String? = null,
    var navn: String? = null
)
